### PR TITLE
Update security-compliance-overview.md

### PR DESCRIPTION
**Pull request by Leicham**
copy edits

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/security-compliance-overview.md [hunk 1]
```diff
@@ -82,29 +82,29 @@ Secure Score helps organizations:
...
 
-Microsoft Teams is supported separately as a cloud app in Azure Active Directory conditional access policies. Conditional access policies that are set for the Microsoft Teams cloud app apply to Microsoft Teams when a user signs in. However, without the correct policies on other apps like Exchange Online and SharePoint, users may still be able to access those resources directly. For more information about setting up a conditional access policy in the Azure portal, see [Azure Active Directory Quickstart](/azure/active-directory/active-directory-conditional-access-azure-portal-get-started).
+Microsoft Teams is supported separately as a cloud app in Azure Active Directory conditional access policies. Conditional Access policies set for the Microsoft Teams cloud app apply only to Microsoft Teams when a user signs in. However, without the correct policies on other apps like Exchange Online and SharePoint, users may still be able to access those resources directly. For more information about setting up a conditional access policy in the Azure portal, see [Azure Active Directory Quickstart](/azure/active-directory/active-directory-conditional-access-azure-portal-get-started).
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md:85]:
```suggestion
Microsoft Teams is supported separately as a cloud app in Azure Active Directory Conditional Access policies. Conditional Access policies set for the Microsoft Teams cloud app apply only to Microsoft Teams when a user signs in. However, without the correct policies on other apps like Exchange Online and SharePoint, users may still be able to access those resources directly. For more information about setting up a Conditional Access policy in the Azure portal, see [Azure Active Directory Quickstart](/azure/active-directory/active-directory-conditional-access-azure-portal-get-started).
```

File: Teams/security-compliance-overview.md [hunk 2]
```diff
@@ -82,29 +82,29 @@ Secure Score helps organizations:
...
 
 ## Compliance
 
-Teams has a wide range of information to help you with compliance areas, including communication compliance for channels, chats, and attachments, retention policies, Data Loss Protection (DLP), eDiscovery and legal hold for channels, chats and files, audit log search, as well as mobile application management with Microsoft Intune. We've provided some information on all these topics below, and you can go to the [Microsoft 365 compliance center](https://compliance.microsoft.com) to manage these settings.
+Teams has a wide range of options for compliance controls, including communication compliance, retention policies, Data Loss Protection (DLP), eDiscovery, and legal hold for channels, chats, and files. Additional controls include the audit log search, mobile application management and mobile device management with Microsoft Intune/Microsoft Endpoint Manager, and many more options. We've provided some information on all these topics below. You can go to the [Microsoft 365 compliance center](https://compliance.microsoft.com) to manage these settings.
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md:93]:
```suggestion
Teams has a wide range of options for compliance control, including communication compliance, retention policies, Data Loss Protection (DLP), eDiscovery, and legal hold for channels, chats, and files. Additional controls include the audit log search, mobile application management and mobile device management with Microsoft Intune/Microsoft Endpoint Manager, and many more options. We've provided some information on all these topics below. You can go to the [Microsoft 365 compliance center](https://compliance.microsoft.com) to manage these settings.
```

File: Teams/security-compliance-overview.md [hunk 3]
```diff
@@ -82,29 +82,29 @@ Secure Score helps organizations:
...
 
 ### Information Barriers
 
-Information barriers are policies put in-place by Teams administrators to do things like keep people or groups from communicating with one another (when there is no business need for them to do so, or a regulatory reason to block them from doing so), and it also allows you to set policies relating to things like lookups and eDiscovery (covered below). These policies can impact users in 1:1 chats, group chats, or at a team-level. The Information Barrier feature is available in the public cloud and starting January 2021 it has been rolled out to the GCC cloud.
+Information barriers are policies put in place by Teams Administrators to do things like keep people or groups from communicating with one another, when there is a business need for them or a regulatory reason to block them from doing so. It also allows you to set policies relating to things like lookups and eDiscovery (covered below). These policies can impact users in 1:1 chats, group chats, or at a team level. The Information Barrier feature is available in the public and GCC clouds.
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md:97]:
```suggestion
Information barriers are policies put in place by Teams administrators to do things like keep people or groups from communicating with one another when there is a business need for them or a regulatory reason to block them from doing so. It also allows you to set policies relating to things like lookups and eDiscovery (covered below). These policies can impact users in 1:1 chats, group chats, or at a team level. The Information Barrier feature is available in the public and GCC clouds.
```

File: Teams/security-compliance-overview.md [hunk 4]
```diff
@@ -82,29 +82,29 @@ Secure Score helps organizations:
...
+Communication compliance in Microsoft 365 allows you to add users to inscope policies that can be configured to examine Microsoft Teams communications for offensive language, sensitive information, and information related to internal and regulatory standards. Chat communications and associated attachments in both public and private Teams channels, individual chats, and attachments can be scanned to help minimize communication risks in your organization. For more information on how you can configure policies to help you detect, capture, and take action for inappropriate Teams communications, see [Communication compliance in Microsoft 365](/microsoft-365/compliance/communication-compliance).
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md:103]:
```suggestion
Communication compliance in Microsoft 365 allows you to add users to in-scope policies that can be configured to examine Microsoft Teams communications for offensive language, sensitive information, and information related to internal and regulatory standards. Chat communications and associated attachments in both public and private Teams channels, individual chats, and attachments can be scanned to help minimize communication risks in your organization. For more information on how you can configure policies to help you detect, capture, and take action for inappropriate Teams communications, see [Communication compliance in Microsoft 365](/microsoft-365/compliance/communication-compliance).
```

File: Teams/security-compliance-overview.md [hunk 5]
```diff
@@ -118,7 +118,7 @@ For information on Data Loss Prevention in Teams, please review [DLP for Microso
 
 ### eDiscovery
 
-Electronic discovery, or eDiscovery, is the electronic aspect of identifying, collecting and producing electronically stored information (ESI) in response to a request for production in a law suit or investigation. Capabilities include case management, preservation, search, analysis, and export of Teams data. This includes chat, messaging and files, meeting and call summaries. For Teams meetings and Calls, a summary of the events that happened in the meeting and call are created and made available in eDiscovery.
+Electronic discovery, or eDiscovery, is the electronic aspect of identifying, collecting, and producing electronically stored information (ESI) in response to a request for production in a lawsuit or investigation. Capabilities include case management, preservation, search, analysis, and export of Teams data. This includes chat, messaging and files, meeting and call summaries. For Teams meetings and calls, a summary of the events that happened in the meeting and call is created and made available in eDiscovery.
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md:121]:
```suggestion
Electronic discovery, or eDiscovery, is the electronic aspect of identifying, collecting, and producing electronically stored information (ESI) in response to a request for production in a lawsuit or investigation. Capabilities include case management, preservation, search, analysis, and export of Teams data. This includes chat, messaging and files, and meeting and call summaries. For Teams meetings and calls, a summary of the events that happened in the meeting and call is created and made available in eDiscovery.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Leicham, kindly commit the minor suggested changes. Many thanks.

**Comment by Leicham** [comment]
> Copy review - @Leicham, kindly commit the minor suggested changes. Many thanks.

done. Thank you, Johan!

**Comment by JohanFreelancer9** [comment]
@Leicham, thank you for committing the suggested changes, but I notice that you have now closed the PR. Was that intentional?

**Comment by Leicham** [comment]
It was accidental!

Sent from my iPhone

On Oct 3, 2021, at 11:55 PM, JohanFreelancer9 ***@***.***> wrote:

﻿

@Leicham<https://github.com/Leicham>, thank you for committing the suggested changes, but I notice that you have now closed the PR. Was that intentional?

**Pull request closed without merging**